recipes/r-rmumps/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set version = '5.2.1-7' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rmumps
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/rmumps_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/rmumps/rmumps_{{ version }}.tar.gz
+  sha256: cebec8e190a4df9356f2e78372d9826bbde04b6c7fe417225cd7c04ea1f5da3e
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ compiler('fortran') }}        # [not win]
+    - {{ compiler('m2w64_fortran') }}  # [win]
+    - {{posix}}filesystem        # [win]
+    - {{posix}}make
+    - {{posix}}sed               # [win]
+    - {{posix}}coreutils         # [win]
+    - {{posix}}zip               # [win]
+  host:
+    - r-base
+    - r-rcpp >=0.12.0
+    - libblas
+    - liblapack
+  run:
+    - r-base
+    - {{native}}gcc-libs         # [win]
+    - r-rcpp >=0.12.0
+
+test:
+  commands:
+    - $R -e "library('rmumps')"           # [not win]
+    - "\"%R%\" -e \"library('rmumps')\""  # [win]
+
+about:
+  home: http://mumps.enseeiht.fr/, https://github.com/sgsokol/rmumps/
+  license: GPL-2
+  summary: 'Some basic features of MUMPS (Multifrontal Massively Parallel sparse direct Solver)
```

Comment:
This summary is too long. Please rename this summary to `description`, and put something shorter for summary, e.g.:
```
summary: Multifrontal Massively Parallel sparse direct Solver
```

Reply:
Done. BTW, thanks for a prompt review.